webpack/assets/javascripts/react_app/components/Layout/components/TaxonomySwitcher/TaxonomyDropdown.js

```diff
@@ -81,7 +90,14 @@ const TaxonomyDropdown = ({ taxonomyType, currentTaxonomy, taxonomies }) => {
   return (
     <ContextSelector
       id={id}
-      toggleText={currentTaxonomy || anyTaxonomyText}
+      toggleText={
+        currentTaxonomy || (
+          <div>
+            <GlobeIcon style={{ marginRight: '5px' }} />
+            {anyTaxonomyText}
```

Comment:
Since we use the same icon and text in two places, can we store this whole JSX structure in a const and use it in the two places?